test/induction/inductive-implicits-bench.scala

```diff
@@ -0,0 +1,619 @@
+// Compiled with ./build/pack/bin/scalac -J-Xss4M -J-Xmx2G test/induction/inductive-implicits-bench.scala
```

Comment:
let's move this to our compiler benchmark project. I don't want tests that are not run in CI -- they will just bitrot.

Reply:
https://github.com/scala/compiler-benchmark

Reply:
Got the project, unsure of the form in which to include it. Currently asking [here](https://gitter.im/scala/compiler-benchmark?at=5ae89db0f2d2d53704554cdf) ... pointers welcome.

Reply:
Done ...